frontend/src/pages/intake-form/components/FormStepper.tsx

```diff
@@ -1,23 +1,43 @@
 import { Tab } from '@headlessui/react';
 import clsx from 'clsx';
-import type { FormTab } from '../constants/types';
+import type { FormTab, IntakeFormValues } from '../constants/types';
+import type { FormikErrors } from 'formik';
+import { useFormikContext } from 'formik';
+import { CheckIcon } from '@/components/ui/Icons';
 
 export const FormStepper = ({
   tab,
-  handleClickTab,
   index,
   formTabs,
+  stepErrors,
+  completedSteps,
+  handleValidateLastStep,
+  handleValidateCurrentStep,
 }: {
   tab: FormTab;
-  handleClickTab: (index: number) => void;
   index: number;
   formTabs: FormTab[];
+  stepErrors?: number[] | null;
+  completedSteps?: number[] | null;
+  handleValidateLastStep: (
+    validateForm: () => Promise<FormikErrors<IntakeFormValues>>,
+    index: number,
+  ) => Promise<void>;
+  handleValidateCurrentStep: (
+    validateForm: () => Promise<FormikErrors<IntakeFormValues>>,
+    index: number,
+  ) => Promise<void>;
 }) => {
+  const { validateForm } = useFormikContext<IntakeFormValues>();
+
```

Comment:
Not sure where it is, but we probably want to disallow clicking on "Complete" (last step)